src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -744,6 +757,29 @@ public ResponseEntity<Mono<Void>> updateLoadModification(@PathVariable("studyUui
                 .then(studyService.updateEquipmentModification(studyUuid, modifyLoadAttributes, ModificationType.LOAD_MODIFICATION, nodeUuid, modificationUuid)));
     }
 
+    @PutMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/modifications/{typeModification}/{modificationUuid}")
+    @Operation(summary = "update a generator modification in the study network")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The generator modification has been updated")})
+    public ResponseEntity<Mono<Void>> updateGeneratorModification(@PathVariable("studyUuid") UUID studyUuid,
```

Comment:
Rename method to updateEquipmentModification since request is generic. Don't forget to modify Operation and ApiReponses either.

Reply:
done